static/app/views/preprod/utils.tsx

```diff
@@ -0,0 +1,49 @@
+import {BuildDetailsArtifactType, type Platform} from 'sentry/views/preprod/types';
+
+export function formatBytes(bytes: number, isSi = false) {
+  if (bytes === 0) return '0 B';
+  const k = isSi ? 1000 : 1024;
+  const sizes = isSi ? ['B', 'KB', 'MB', 'GB', 'TB'] : ['B', 'KiB', 'MiB', 'GiB', 'TiB'];
+  const i = Math.floor(Math.log(bytes) / Math.log(k));
+  return parseFloat((bytes / Math.pow(k, i)).toFixed(2)) + ' ' + sizes[i];
+}
+
+export function getPlatformFromArtifactType(
```

Comment:
Will remove in a follow up following https://github.com/getsentry/sentry/pull/95854/files#r2216408619